torchbenchmark/models/hf_Bert/__init__.py

```diff
@@ -46,6 +55,8 @@ def train(self, niter=3):
             loss = outputs.loss
             loss.backward()
             self.optimizer.step()
+            if self.device == 'lazy':
```

Comment:
Can we place lazy_tensor related code in a separate python file so that if we need to change it in the future we don't need to change every model's code? E.g. in `torchbenchmark/util/lazy_tensor.py` we have a function `lazy_step()` that does this. @alanwaketan

Reply:
I am not relying on mark_step in this file.  In fact on my own branch I deleted this change.

I wrote another script (lazy_bench.py) which iters over torchbench models and calls the sync functions at the right times.

There will be some small changes needed in main torchbench model code, only for particular models where some hardcoded 'cuda' or 'cpu' assumption is baked in and we need to loosen it.

Reply:
Addressed, can you help take another look?